src/sentry/api/endpoints/organization_events_anomalies.py

```diff
@@ -62,7 +62,8 @@ def _format_historical_data(self, data) -> list[TimeSeriesPoint] | None:
 
         formatted_data: list[TimeSeriesPoint] = []
         for datum in data:
-            ts_point = TimeSeriesPoint(timestamp=datum[0], value=datum[1].get("count", 0))
```

Comment:
.get defaults to 0 if the `count` key is missing from the dict, but if the dict has `count = None` then it will still set `value` to None. So we need to convert those cases to 0 as well.